client/src/modules/dashboard/Chapters/components/ChapterForm.tsx

```diff
@@ -70,7 +70,7 @@ const fields: Fields[] = [
     type: 'text',
   },
   {
-    key: 'image_url',
+    key: 'banner_url',
     label: 'Image Url',
```

Comment:
Shouldn't the label also be changed? Otherwise changing field name is a bit pointless.

Reply:
for some reason, I thought I have changed in cypress, and it wasn't complaining so I changed it in the main repo.

But I didn't change it in, either. 😅